DEV: flesh out FT.INFO page (#444)
redis/docs: content/commands/ft.info/index.md

| `offset_vectors_sz_mb` | The memory used by the offset vectors, which store positional information for terms in documents. |
| `doc_table_size_mb` | The memory used by the document table, which contains metadata about each document in the index. |
| `sortable_values_size_mb` | The memory used by sortable values, which are values associated with documents and used for sorting purposes. |
| `key_table_size_mb` | The memory used by the key table, which stores the mapping between document IDs and Redis keys. |
| `geoshapes_sz_mb` | The memory used by GEO-related fields. |
| `records_per_doc_avg` | The average number of records (including deletions) per document. |
| `bytes_per_record_avg` | The average size of each record in bytes. |
| `offsets_per_term_avg` | The average number of offsets (position information) per term. |
| `offset_bits_per_record_avg` | The average number of bits used for offsets per record. |

### Indexing-related statistics

| Statistic | Definition |
|:---       |:---        |
| `hash_indexing_failures` | The number of failures encountered during indexing. |
| `total_indexing_time` | The total time is taken for indexing in seconds. |

> | `indexing` | Indicates whether the index is currently being indexed. |

Reviewer: Looks a bit odd like that. Maybe "Indicates whether the data is currently being indexed" is another option.
```suggestion
| `indexing` | Indicates whether the index is currently being generated. |
```

> | `percent_indexed` | The percentage of the index that has been successfully indexed. |

Reviewer: ```suggestion
| `percent_indexed` | The percentage of the index that has been successfully generated. |
```

| `number_of_uses` | The number of times the index has been used. |

> | `cleaning` | DWD need def'n |

Author: ```suggestion
| `cleaning` | The index deletion flag. A value of `1` is an indication that index deletion is in progress. |
```

> | `cleaning` | The index deletion flag. A value of `1` is an indication that index deletion is in progress. |

Author: ```suggestion
| `cleaning` | The index deletion flag. A value of `1` indicates index deletion is in progress. |
```

### Garbage collection statistics

| Statistic | Definition |
|:---       |:---        |
| `bytes_collected` | The number of bytes collected during garbage collection. |
| `total_ms_run` | The total time in milliseconds spent on garbage collection. |
| `total_cycles` | The total number of garbage collection cycles. |
| `average_cycle_time_ms` | The average time in milliseconds for each garbage collection cycle. The value `nan` indicates that the average cycle time is not available. |
| `last_run_time_ms` | The time in milliseconds taken by the last garbage collection run. |

> The next two GC-related fields are relevant in scenarios where simultaneous changes occurred in the same memory area by both the parent process and the child process, resulting in the parent discarding these changes.

Reviewer: ```suggestion
The next two GC-related fields are relevant in scenarios where simultaneous changes occurred in the same memory area for both the parent process and the child process, resulting in the parent discarding these changes.
```
